pandas/tests/io/excel/test_style.py

```diff
@@ -205,3 +205,21 @@ def custom_converter(css):
 
         with contextlib.closing(openpyxl.load_workbook(path)) as wb:
             assert wb["custom"].cell(2, 2).font.color.value == "00111222"
+
+
+def test_styler_to_s3(self, s3_resource, s3so):
+    import time
```

Comment:
Can you move this import to the top of the file

Reply:
Also add the issue number here as a comment (not at the top of the file!): `# GH#46381`